core/examples/yaml/processSummaryLocalResource.yaml

```diff
@@ -0,0 +1,10 @@
+---
+allOf:
+  - $ref: 'recordCommonProperties.yaml'
+  - $ref: 'https://schemas.opengis.net/ogcapi/processes/part1/1.0/openapi/schemas/processSummary.yaml'
```

Comment:
Is pointing to an OGC API - Processes schema intentional?

Reply:
Yes, this is an example of extents the `/processes/{processId}` endpoint to make is a local resources endpoint.